sdk/digitaltwins/digital-twins-core/test/public/testComponents.spec.ts

```diff
@@ -71,24 +71,24 @@ describe("DigitalTwins Components - read, update and delete operations", () => {
   let client: DigitalTwinsClient;
   let recorder: Recorder;
 
-  beforeEach(async function() {
+  beforeEach(async function (this: Mocha.Context) {
     const authentication = await authenticate(this);
     client = authentication.client;
     recorder = authentication.recorder;
   });
 
-  afterEach(async function() {
+  afterEach(async function () {
     await recorder.stop();
   });
 
   async function deleteModels(): Promise<void> {
     try {
       await client.deleteModel(MODEL_ID);
-    } catch (Exception) {}
+    } catch (Exception) {/* ignored */ }
```

Comment:
Any reason why we cannot have the error bubble up?
Am guessing this is part of some cleanup that is done before each test.
Wouldnt it be good to know if the clean up failed?

Same feedback applies to other places where we are catching and then ignoring the errors

Reply:
You are correct. I'll add some logging for those failures.